modules/amps/meta.yml

```
@@ -0,0 +1,66 @@
+name: amps
+description: Post-processing script of the MaltExtract component of the HOPS package
+keywords:
+  - malt
+  - MaltExtract
+  - HOPS
+  - amps
+  - alignment
+  - metagenomics
+  - ancient DNA
+  - aDNA
+  - palaeogenomics
+  - archaeogenomics
+  - microbiome
+  - authentication
+  - damage
+  - edit distance
+  - post Post-processing
+  - visualisation
+tools:
+  - amps:
+      description: Post-processing script of the MaltExtract tool for ancient metagenomics
+      homepage: "https://github.com/rhuebler/HOPS"
+      documentation: "https://github.com/keyfm/amps"
+      tool_dev_url: "https://github.com/keyfm/amps"
+      doi: "10.1186/s13059-019-1903-0"
+      licence: ['GPL >=3']
+
+input:
+  - maltextract_results:
+      type: directory
+      description: MaltExtract output directory
+      pattern: "results/"
```

The directory can not be named otherwise? If yes:
```suggestion
      pattern: "*"
```

---

Nope, specificly called `results/` via the prevoius tool.

---

But it can be changed with `-o(utput)/myOutDirectory/', anyway not very relevant since is in the yml file :smiley:

---

Aaaand you're totally right, I'm remembering an old version of MaltExtract 🙄🤦🏼‍♀️